### ISSUE TITLE
KML in kaart niet op juiste plek

### ISSUE THREAD

**Issue by gerardnienhuis** [IDgis/geoportaal-test#149]
Het lijkt er op dat de KML op een andere plek op de kaart wordt gezet als de shape en gml die kan worden gedownload: lijkt op een probleem met projecties te maken

In de bijgevoegde kaart is groen correct, en blauw de verschoven kml download (er bovenop getekend)
![kml_te_ver_oostelijk](https://cloud.githubusercontent.com/assets/9005444/14796476/bb92c346-0b2d-11e6-91a7-fc47337c1c26.jpg)

gebruikte bestanden om te controleren:

Bestandsnaam: Weidevogel- en ganzengebieden
Alternatieve titel: B4.weidevog_ganzen_polygon (b4\b47)
Download: https://acc.geoportaaloverijssel.nl/download/b8c5f4d2-b4f5-45a7-9dbd-42df6cf873f5
OGC:WMS: http://acc-services.geodataoverijssel.nl/geoserver/B47_flora_en_fauna/wms
OGC:WFS: http://acc-services.geodataoverijssel.nl/geoserver/B47_flora_en_fauna/wfs

Bestandsnaam: Aanwijzing foerageergebieden voor ganzen en smienten
Alternatieve titel: B4.gansfoer_polygon (b4/b47)
Download: https://acc.geoportaaloverijssel.nl/download/cc887358-bd56-4f5d-a39c-b5c4bf28ab6a
OGC:WMS: http://acc-services.geodataoverijssel.nl/geoserver/B47_flora_en_fauna/wms
OGC:WFS: http://acc-services.geodataoverijssel.nl/geoserver/B47_flora_en_fauna/wfs

**Comment by HermanAssink** [comment]
Inderdaad, vermoedelijk a.g.v. verschil in projecties. Moeten we uitzoeken

**Comment by HermanAssink** [comment]
Gewenst is KML in RD. Vermoedelijk is de transformatie onnauwkeurig.

**Comment by gerardnienhuis** [comment]
epsg code
rd: 28992  (voor alle downloadformaten), met uitzondering van google KML
google 3857
komt in helptekst

**Comment by HermanAssink** [comment]
Zoeken we eerst beter uit.

**Comment by HermanAssink** [comment]
@copierrj : Is er iets mis in onze programmatuur of is er een andere oorzaak?

**Comment by copierrj** [comment]
Is een bug in GeoServer of in QGIS.

**Comment by copierrj** [comment]
KML gebruikt WGS-84

**Comment by gerardnienhuis** [comment]
Dit is getest in ArcGIS (arcmap). Het blijkt dat zelfs als de KML wordt geconverteerd en de projectie daarna van WGS84 wordt omgezet naar RD, hier een verschuiving te zien is: in het voorbeeldplaatje is de KML 75 meter te veel in het oosten geplaatst. Kan het zijn dat de geoserver instellingen voor KML niet volledig juist zijn?
![kml](https://cloud.githubusercontent.com/assets/9005444/15319334/9fb77a32-1c2a-11e6-9626-5f0db1ff2a8f.PNG)

**Comment by HermanAssink** [comment]
KML is per definitie in WGS-84 / EPSG:4326. Wanneer je bijv. een KML van reactievaartklassen opent in Google Earth blijkt deze goed te passen:

![image](https://cloud.githubusercontent.com/assets/2979044/15429865/1b7985dc-1ea3-11e6-9f2a-6b7ad203cd14.png)

**Comment by gerardnienhuis** [comment]
Ik begrijp niet waardoor ik nog steeds deze verschillen blijf houden.

Ik heb opnieuw getest met het bestand "afwatering" uit het externe geoportaal en zie nog steeds verschuiving van zo'n 60 meter, ook in Google Earth (getest door eerst download uit te voeren, daarna met "bestand open" het bestand in Google Earth in te lezen).

zie plaatje: 

![goodle_earth_60_meter](https://cloud.githubusercontent.com/assets/9005444/15737554/eb4fd3c6-28a6-11e6-94be-ea82b69612c2.jpg)

**Comment by gerardnienhuis** [comment]
Overigens, ook bij het bestand "'B2_Toekenning_recreatievaartklasse_vaarwegen" is de verschuiving van 60 meter bij mij wel te zien, als je wat verder gaat inzoomen: het lijkt op een regelmatig terugkomend probleem: KML laat tov werkelijkheid een verschuiving van 60 meter zien, naar het oosten (noord-oosten?). Zou het een instelling in de software van de publicatieomgeving kunnen zijn, b.v. Geoserver?

zie plaatje: 

![goodle_earth_60_meter_recreatievaart](https://cloud.githubusercontent.com/assets/9005444/15737856/a1f9f128-28a8-11e6-9c74-d601c14429d3.jpg)

**Comment by NicoEilers** [comment]
Het is bekend dat de google projectie niet overal even nauwkeurig is. Voorstel is dit tzt keer verder uit te zoeken en nu te laten voor wat het is. Komt wel extra tekst bij, in de helppagina's alvast. Issue wordt nog niet gesloten, maar kan bij versie 1.1. worden opgepakt.

**Comment by gerardnienhuis** [comment]
effect is ook heel goed te zien bij lijnbestand van gemeenten:
vergelijk shape of gml met kml in qgis, levert verschuiving (foutief) van kml op van meer dan 65 meter, zie plaatje. 
bestand wat in dit voorbeeld is gebruikt is: 

https://www.geoportaaloverijssel.nl/metadata/dataset/b410f7c4-fbc0-48b9-877d-fa0a32130a4d
Bestandsnaam: Grenzen van alle Nederlandse gemeenten
Download: https://download.geoportaaloverijssel.nl:443/download/b410f7c4-fbc0-48b9-877d-fa0a32130a4d

![kml_verschuiving](https://cloud.githubusercontent.com/assets/9005444/19012295/40a49082-87b1-11e6-9060-bf8835a4c073.jpg)
